main.go

```diff
@@ -15,13 +16,14 @@ const (
 )
 
 type Parcel struct {
-	Number    int
-	Client    int
-	Status    string
-	Address   string
-	CreatedAt string
+	Number     int
+	Client     int
+	Status     string
+	Address    string
+	Created_At string
```

Comment:
В Го принято называть переменные используя CamelCase
```suggestion
	CreatedAt string
```